lib/improver/tests/helper_functions_ensemble_calibration.py

```diff
@@ -71,6 +71,36 @@ def set_up_temperature_cube():
     return set_up_cube(data, "air_temperature", "K")
 
 
+def set_up_spot_cube(data, phenomenon_standard_name, phenomenon_units):
```

Comment:
Yes, I think this is the only place where I define a "locnum" coordinate. Unit tests for spot forecasts already exist, but yes, these could be reviewed as part of #48 .